fastlane/lib/fastlane/actions/update_project_provisioning.rb

```diff
@@ -63,6 +64,13 @@ def self.run(params)
               next
             end
 
+            unless codesigning_identity.nil?
+              unless build_configuration.build_settings["CODE_SIGN_IDENTITY[sdk=iphoneos*]"].nil?
+                build_configuration.build_settings["CODE_SIGN_IDENTITY[sdk=iphoneos*]"] = codesigning_identity
+              end
+              build_configuration.build_settings["CODE_SIGN_IDENTITY"] = codesigning_identity
```

Comment:
Can we change the code in this `unless` to be...
```rb
codesign_build_settings_keys = build_configuration.build_settings.keys.select { |key| key.to_s.match(/CODE_SIGN_IDENTITY.*/) }
codesign_build_settings_keys.each do |setting|
  build_configuration.build_settings[setting] = codesigning_identity
end
```
so that it works with other platforms and not just iOS? Inspired by https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/automatic_code_signing.rb#L46-L49

Reply:
Great idea